docs/src/user-manual/ingest/index.md

```diff
@@ -0,0 +1,38 @@
+# Ingest
+
+This section of the User manual covers the [Ingest](../glossary.md#ingest)
+domain functionality.
```

Comment:
Would remove and just start at the following paragraph

Reply:
Removed